test/w3c/index.js

```diff
@@ -59,3 +59,138 @@ w3cTest("html/infrastructure/urls/terminology-0/document-base-url.html");
 w3cTest("html/syntax/serializing-html-fragments/outerHTML.html");
 w3cTest("html/webappapis/atob/base64.html");
 
+
```

Comment:
this is AMAZING. You are the best person ever. Thank you so much for getting all these passing!!!!